server/handlers/clients.js

```diff
@@ -5,9 +5,10 @@ const { configHelpers, clientFactory } = require('../utils');
 const { getDefaultConfig, testConfigOptions, getConfig } = configHelpers;
 
 // utility to return basic info about a client based on its config
-function getClientInfo(config) {
+function getClientInfo(config, req) {
```

Comment:
I don't think that the req object should go into this function, it makes the abstraction strange because it doesn't act on the req, it acts on `clientHealth` which it shouldn't care where that comes from

Reply:
yes! good call. I think I did this at one stage because I was also using the logger, but this is functionally just a utility for getting the data in a certain shape, definitely irrelevant to what it's doing now though.

Reply:
Fixed.